charts/rbacdefinitions/Chart.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: rbacdefinitions
```

can we have dash in the name please?
```suggestion
name: rbac-definitions
```

---

yeah was bugging me too during the whole creation. glad you think the same. I am changing this later.

---

i was even considering `rbac-manager-definitions` but it might get a tad to long with that.